test/library/packages/UnitTest/GetTest/Primitive_Get_multiple.chpl

```diff
@@ -19,7 +19,7 @@ proc test_squared(test: borrowed Test) throws {
 }
 
 
-var test: owned Test;
+var test: owned Test = new owned Test();
```

Comment:
In cases like this, I usually drop the type annotation, leaving just `var test = new ...;`

OK to leave as-is.